RecoTracker/LST/python/lstProducerTask_cff.py

```diff
@@ -0,0 +1,8 @@
+import FWCore.ParameterSet.Config as cms
+
+from RecoTracker.LST.lstProducer_cfi import lstProducer
+
+from RecoTracker.LST.lstModulesDevESProducer_cfi import lstModulesDevESProducer
+
+# not scheduled task to get the framework to hide the producer
```

Comment:
@slava77 Not sure whether you intended to add anything more in your #115 PR, but, if you do, could you also remove this comment? I forgot to do it while updating this file.

Reply:
I started picking up other stuff at first, but then made a long-range change and reverted back to just a technically required change. 
So, I didn't really intend to, but I can, if you prefer.

We still need an update in python files elsewhere to resolve the conflicts; could be added there.

Reply:
Indeed, I noticed the conflicts only afterwards, and it can be added there. Let me give that a try now.